lib/transport/data_validation/queries/find_feed_version.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Transport.DataValidation.Queries.FindFeedVersion do
+  @moduledoc """
+  Query for finding a feed version.
+
+  ## Examples
+
+      iex> %{project: %Project{id: "1"}, latest_version_id: "export-quotidien-au-format-gtfs-du-reseau-de-transport-lignes-d-azur-20180315T162637Z-16f8b757-1427-42d9-937f-a89633616334.zip"}
```

Comment:
For readability, wouldn't it work just as well with a shorter dummy `latest_version_id` ?